plugins/backend/rag-ai-backend/src/service/LlmService.ts

```diff
@@ -51,6 +52,7 @@ export class LlmService {
     const prompt = `Human:\n${this.prompts.prefixPrompt(
       promptEmbeddings,
     )}\n ---\n${this.prompts.suffixPrompt(query)}\nAssistant:`;
-    return await this.model.invoke(prompt);
+    const response = await this.model.invoke(prompt);
+    return typeof response === 'string' ? response : response.content;
```

Comment:
Are we able to determine the type of response shape more specifically here?

Reply:
Yes: `this.model._modelType()` returns `base_chat_model`, but then we lose TypeScript's automatic type inference. That's why I didn't end up using it.

Reply:
Aight, grand. Let's ship